main/src/main/res/values/strings.xml

```diff
@@ -2411,6 +2411,14 @@
     <string name="command_rename_list_result">Renamed list</string>
     <string name="command_delete_list_result">Deleted list</string>
     <string name="command_unique_list_result">Deleted caches from other lists</string>
+    <plurals name="fav_points_remaining_oc">
+        <item quantity="zero">I want to recommend this cache (%d remaining)</item>
```

Comment:
Maybe a better/shorter wording would be `Recommend this cache (%d remaining)`. This is also similar to the favorite wording.